src/briefcase/macos.py

```diff
@@ -58,6 +62,14 @@ def launcher_script_location(self):
         return macos_dir
 
     def build_app(self):
+        print("Building DMG file...")
+        dmg_name = self.formal_name + '.dmg'
+        dmg_path = os.path.join(os.path.abspath(self.dir), dmg_name)
+        settings = {'files': [self.app_location],
+                    'symlinks': {'Applications': '/Applications'}}
+        dmgbuild.build_dmg(filename=dmg_path,
+                           volume_name=self.formal_name, settings=settings)
+
         return True
 
```

Comment:
At column 80: (E501) line too long (84 > 79 characters)

Reply:
From the look of it `autopep8`s default configuration for line length is 79 chars. Maybe Brutus counts the whitespaces as well?